### PR TITLE
Add documentation for Point oauth2

### PR DESCRIPTION
**Pull request by fredrike**
## Proposed change

https://github.com/home-assistant/core/pull/118243 implements use of HA's internal oauth2 authentication for Point this PR documents how to configure Point.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/118243
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/point.markdown [hunk 1]
```diff
@@ -19,48 +19,44 @@ ha_platforms:
...
-Once Home Assistant is started, a configurator will pop up asking you to Authenticate your Point account via a link. When you follow the link and click on **Accept** you will be redirected to the `callback url` and the Point integration will be automatically configured and you can go back to the original dialog and press **Submit**.
+- [Device types](#device-types)
+  - [Alarm](#alarm)
+  - [Binary sensor](#binary-sensor)
+    - [Automation example](#automation-example)
+    - [Webhook events](#webhook-events)
+  - [Sensor](#sensor)
 
-There is currently support for the following device types within Home Assistant:
+## Get Minut Point application credentials
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
## Prerequisites

Before adding the integration to Home Assistant, you need to get Minut Point application credentials.
```

File: source/_integrations/point.markdown [hunk 2]
```diff
@@ -19,48 +19,44 @@ ha_platforms:
...
 
-- [Alarm](#alarm)
-- [Binary sensor](#binary-sensor)
-- [Sensor](#sensor)
+Navigate to the [API-client | Minut](https://web.minut.com/settings/api-clients) dashboard and **Create client**:
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
1. Navigate to the [API-client | Minut](https://web.minut.com/settings/api-clients) dashboard and **Create client**:
```

File: source/_integrations/point.markdown [hunk 3]
```diff
@@ -19,48 +19,44 @@ ha_platforms:
...
 
-### Configuration
+- Enter a **Name** for your client (this is just an identifier).
+- Enter `https://my.home-assistant.io/redirect/oauth` in the **Redirect URI** field.
+- Get the **ClientID** and **ClientSecret** for the new client, you need them to complete the integration setup in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
  - Enter a **Name** for your client (this is just an identifier).
  - Enter `https://my.home-assistant.io/redirect/oauth` in the **Redirect URI** field.
2. Get the **ClientID** and **ClientSecret** for the new client and store them in a safe place. You need them to complete the integration setup in Home Assistant.
```

File: source/_integrations/point.markdown [hunk 4]
```diff
@@ -19,48 +19,44 @@ ha_platforms:
...
-{% include integrations/restart_ha_after_config_inclusion.md %}
+<div class='note'>
 
-```yaml
-# Example configuration.yaml entry
-point:
-  client_id: CLIENT_ID
-  client_secret: CLIENT_SECRET
-```
+If you are a kickstarter backer you need to send an email to hello@minut.com to retrive **ClientID** and **ClientSecret**, don't forget to mention that the **Redirect URI** should be `https://my.home-assistant.io/redirect/oauth`.
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
If you are a Kickstarter backer, you need to send an email to hello@minut.com to retrieve the **ClientID** and **ClientSecret**. Don't forget to mention that the **Redirect URI** should be `https://my.home-assistant.io/redirect/oauth`.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweaks

**Comment by fredrike** [comment]
Thanks for the suggestions @c0ffeeca7!

File: source/_integrations/point.markdown [hunk 5]
```diff
@@ -19,48 +19,46 @@ ha_platforms:
...
-- [Sensor](#sensor)
+Before adding the integration to Home Assistant, you need to get Minut Point application credentials.
 
-### Configuration
+1. Navigate to the [API-client | Minut](https://web.minut.com/settings/api-clients) dashboard and **Create client**:
 
-To use the Minut Point {% term integration %} in your installation, add it to your {% term "`configuration.yaml`" %} file.
-{% include integrations/restart_ha_after_config_inclusion.md %}
+  - Enter a **Name** for your client (this is just an identifier).
+  - Enter `https://my.home-assistant.io/redirect/oauth` in the **Redirect URI** field.
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
    - Enter a **Name** for your client (this is just an identifier).
    - Enter `https://my.home-assistant.io/redirect/oauth` in the **Redirect URI** field.
```
Got the indents wrong in my previous comment

File: source/_integrations/point.markdown [hunk 6]
```diff
@@ -19,48 +19,46 @@ ha_platforms:
...
-- [Sensor](#sensor)
+Before adding the integration to Home Assistant, you need to get Minut Point application credentials.
 
-### Configuration
+1. Navigate to the [API-client | Minut](https://web.minut.com/settings/api-clients) dashboard and **Create client**:
 
-To use the Minut Point {% term integration %} in your installation, add it to your {% term "`configuration.yaml`" %} file.
-{% include integrations/restart_ha_after_config_inclusion.md %}
+    - Enter a **Name** for your client (this is just an identifier).
+    - Enter `https://my.home-assistant.io/redirect/oauth` in the **Redirect URI** field.
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
   - Enter a **Name** for your client (this is just an identifier).
   - Enter `https://my.home-assistant.io/redirect/oauth` in the **Redirect URI** field.
```

File: source/_integrations/point.markdown [hunk 7]
```diff
@@ -19,48 +19,46 @@ ha_platforms:
...
-client_secret:
-  description: Your Minut Point developer client secret.
-  required: true
-  type: string
-{% endconfiguration %}
+</div>
+
+{% include integrations/config_flow.md %}
 
 # Device types
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
## Device types
```

File: source/_integrations/point.markdown [hunk 8]
```diff
@@ -19,48 +19,46 @@ ha_platforms:
...
-The Point hub enables integration with the [Minut Point](https://minut.com/). To connect with Point, you will have to [sign up for a developer account and have a Pro subscription](https://minut.com/community/developers/) and get a `client_id` and `client_secret` with the `callback url` configured as your Home Assistant URL + `/api/minut`, e.g.,  `http://homeassistant.local:8123/api/minut`. The `client_id` and `client_secret` should be used as below.
+The Point hub enables integration with the [Minut Point](https://minut.com/). There is currently support for the following device types within Home Assistant:
 
-Once Home Assistant is started, a configurator will pop up asking you to Authenticate your Point account via a link. When you follow the link and click on **Accept** you will be redirected to the `callback url` and the Point integration will be automatically configured and you can go back to the original dialog and press **Submit**.
+- [Device types](#device-types)
+  - [Alarm](#alarm)
+  - [Binary sensor](#binary-sensor)
+    - [Automation example](#automation-example)
+    - [Webhook events](#webhook-events)
+  - [Sensor](#sensor)
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
```
This section is duplicated below.

File: source/_integrations/point.markdown [hunk 9]
```diff
@@ -19,48 +19,46 @@ ha_platforms:
...
 ha_integration_type: integration
 related:
-  - docs: /docs/configuration/
-    title: Configuration file
+  - docs: /integrations/application_credentials/
+    title: Application credentials
 ---
 
-The Point hub enables integration with the [Minut Point](https://minut.com/). To connect with Point, you will have to [sign up for a developer account and have a Pro subscription](https://minut.com/community/developers/) and get a `client_id` and `client_secret` with the `callback url` configured as your Home Assistant URL + `/api/minut`, e.g.,  `http://homeassistant.local:8123/api/minut`. The `client_id` and `client_secret` should be used as below.
+The Point hub enables integration with the [Minut Point](https://minut.com/). There is currently support for the following device types within Home Assistant:
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
The Point hub enables integration with the [Minut Point](https://minut.com/).
```

File: source/_integrations/point.markdown [hunk 10]
```diff
@@ -19,48 +19,46 @@ ha_platforms:
...
-{% endconfiguration %}
+</div>
+
+{% include integrations/config_flow.md %}
 
 # Device types
 
 The integration supports the following device types within Home Assistant:
+
+- [Device types](#device-types)
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
```

File: source/_integrations/point.markdown [hunk 11]
```diff
@@ -19,48 +19,46 @@ ha_platforms:
...
   - [Alarm](#alarm)
   - [Binary sensor](#binary-sensor)
+    - [Automation example](#automation-example)
+    - [Webhook events](#webhook-events)
```
Comment (c0ffeeca7) [inline source/_integrations/point.markdown]:
```suggestion
```

**Review by c0ffeeca7 — COMMENTED** [review]
Missed a few last time

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @fredrike :+1:

**Pull request merged**